src/request.js

```diff
@@ -0,0 +1,244 @@
+/******************************************************************************
+ * Licensed Materials - Property of IBM
+ * IBM Bluemix Container Service, 5737-D43
+ * (C) Copyright IBM Corp. 2023 All Rights Reserved.
+ * US Government Users Restricted Rights - Use, duplication or
+ * disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+ ******************************************************************************/
```

Comment:
I recommend using the copyright statement from existing files in this repo (with `2023`) for consistency.